config/initializers/unify_fog_cloud_clients.rb

```diff
@@ -186,4 +186,10 @@ def supported_architectures
     else 'x86_64'
     end
   end
-end
\ No newline at end of file
+end
+
+class Fog::Compute::OpenStack::Real
```

Comment:
Unused method argument - `server_id`. If it's necessary, use `_` or `_server_id` as an argument name to indicate that it won't be used. You can also write as `create_tags_for_vm(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `tags_map`. If it's necessary, use `_` or `_tags_map` as an argument name to indicate that it won't be used. You can also write as `create_tags_for_vm(*)` if you want the method to accept any arguments but don't care about them.